gui/femmt_gui.py

```diff
@@ -645,6 +661,14 @@ def update_annot(ind):
             annot.set_text(text)
             annot.get_bbox_patch().set_alpha(0.4)
 
+    def setup_aut_simulation_type_combobox(self):
```

Comment:
This functionality should be removed.
Please search for the following code line and change it accordingly (removing the transformer)

aut_simulation_type_options = [self.translation_dict['inductor'], self.translation_dict['transformer']]